Nodejs/Product/Nodejs/NpmUI/NpmPackageInstallViewModel.cs

```diff
@@ -53,12 +53,14 @@ internal static class FilterState
         private readonly Timer filterTimer;
         private readonly Dispatcher dispatcher;
         private readonly NpmWorker npmWorker;
+        private readonly Action<int> searchResultCallback;
 
         private bool disposed = false;
 
         public NpmPackageInstallViewModel(
             NpmWorker npmWorker,
-            Dispatcher dispatcher
+            Dispatcher dispatcher,
+            Action<int> searchResultCallback
```

Comment:
It would be slightly more idiomatic to expose this as an event.

Reply:
Changed.